circle.yml

```diff
@@ -1,14 +1,10 @@
 ## Customize the test machine
 machine:
-
   node:
-    version: 8.9.1
+    version: 8.9.4
   timezone:
     America/Los_Angeles # Set the timezone
 
-  php:
```

Comment:
Should we really remove this? Since we don't (currently) test on several versions of PHP, I figure that testing on an old version will at least give us some confidence into our support of older versions. If we remove this, I assume the tests will run on the latest version?

Reply:
The default PHP version is 5.6.17. It doesn't support anything earlier than 5.5.31 as per https://circleci.com/docs/1.0/build-image-trusty/#php